test_scripts/UpdateVideoStreamingCapabilities/001_7_GetCapabilities_with_addVSC_only.lua

```diff
@@ -0,0 +1,42 @@
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0296-Update-video-streaming-capabilities-during-ignition-cycle.md
+--
+-- Description: SDL applies the videoStreamingCapability with only additionalVideoStreamingCapabilities parameter
+--  received from HMI in UI.GetCapabilities response
+--
+-- Preconditions:
+-- 1. SDL and HMI are started
+
+-- Sequence:
+-- 1. SDL requests UI.GetCapabilities()
+-- 2. HMI sends UI.GetCapabilities(videoStreamingCapability) response with only additionalVideoStreamingCapabilities
+--  parameter
+-- SDL does:
+-- - a. apply the videoStreamingCapability with only additionalVideoStreamingCapabilities parameter internally
+-- 3. App registers with 5 transport protocol
+-- 4. App requests GetSystemCapability(VIDEO_STREAMING)
+-- SDL does:
+-- - a. send GetSystemCapability response with videoStreamingCapability that contains
+--    only additionalVideoStreamingCapabilities received from HMI in UI.GetCapabilities response
+---------------------------------------------------------------------------------------------------
+-- [[ Required Shared libraries ]]
+local common = require('test_scripts/UpdateVideoStreamingCapabilities/common')
+
+local vsc = {
+    additionalVideoStreamingCapabilities = {}
```

Comment:
```suggestion
   additionalVideoStreamingCapabilities = {}
```
Please remove redundant a whitespace

Reply:
Done (see 38967ac)